### DISCUSSION TITLE
How to write param`s comments on a request?

### DISCUSSION THREAD

**Discussion by aliuq** [hoppscotch/hoppscotch#1971] [Q&A] [answered]
How to write param`s comments on a request?

**Comment by liyasthomas** [comment]
There's no support to add comments to parameter entries at the moment. We will look into adding this feature if there's enough request and usage for this.

**Reply by aliuq** [reply]
Thanks for your answer @liyasthomas

**Reply by jimingeorge** [reply]
Need support for the above. Please add them or suggest an alternative

**Reply by makkmarci13** [reply]
I'll be also happy if you add this feature :)

**Reply by mstimvol** [reply]
I'm also interested in adding this feature :-)

**Comment by harshgupta-mindit** [comment]
Still waiting this feature to be added.

**Comment by AlbertSu123** [comment]
Please add comments!
<img width="1035" alt="image" src="https://github.com/user-attachments/assets/0206481f-c6a0-44a9-b916-001b8da3d83e">

**Comment by liyasthomas** [comment, marked as answer]
Description field on parameters are now available.

Closing this ticket as the fix is now available in the recent version. Please feel free to reach out if you have any other concerns.

**Reply by keay250** [reply]
Is this feature only available at https://hoppscotch.io? I don't see anything about it when I use Self-Host.

**Reply by liyasthomas** [reply]
Can you make sure you're on the latest release version. This feature is available starting from [v2024.8.0](https://github.com/hoppscotch/hoppscotch/releases/tag/2024.8.0).

**Reply by keay250** [reply]
Oh, I've found the problem, thank you very much!

**Comment by mstimvol** [comment]
I can see it for parameters, but not for request body (e.g. `application/x-www-form-urlencoded` or `multipart/form-data`). Is this intended?

**Reply by mstimvol** [reply]
Any update on request body parameters?

**Reply by 1226151667** [reply]
@liyasthomas Thank you very much for your review. We hope to support the parameter description of this request body, similar to Postman
<img width="2012" height="381" alt="image" src="https://github.com/user-attachments/assets/31908a78-2c11-412f-8760-7dea448fabdd" />
